ui/admin/app/routes/scopes/scope/auth-methods.js

```diff
@@ -179,21 +179,20 @@ export default class ScopesScopeAuthMethodsRoute extends Route {
    */
   @action
   async addStringItem(authMethod, property, value) {
-    const array = authMethod.get(property);
-    array.addObject({ value });
+    const array = [...(authMethod.get(property) ?? []), { value }];
```

Comment:
For readability, could this be an improvement...
```suggestion
    const existing = ...authMethod?.[property]  || [];
    const array = [...existing, { value }];
```

Reply:
Yup I agree, I'll update that in a few places!